pages/your_domain/structural_bioinformatics.md

```diff
@@ -1,36 +1,50 @@
 ---
 title: Structural Bioinformatics
-search_exclude: true
-contributors: [<!---REPLACE THIS with comma separated list of contributors--->]
-page_id: struct info
+contributors: [Gerardo Tauriello, Ian Sillitoe, Nicola Bordin, Christine Orengo, Mihaly Varadi, Sameer Velankar, Jiří Černý]
+page_id: struct bioinfo
 related_pages: 
   your_tasks: []
   tool_assembly: []
+training:
+  - name: Training in TeSS
+    registry: TeSS
+    registry_url: https://tess.elixir-europe.org
+    url: https://tess.elixir-europe.org/search?utf8=%E2%9C%93&q=Structural+Bioinformatics#workflows
 ---
 
-<!--- Domain pages should detail the particular data management challenges of the domain, typically by complementing and extending one or more existing Problem pages.
-In the event that no adequate Problem page exists for a problem that can be generalized across domains, consider first contributing to create one or raising a GitHub issue. However, if a problem is entirely domain specific, then it should be fully detailed within the respective Domain page. --->
-
 ## Introduction
 
-<!--- In this section you should provide a brief overview of the domain from the data management perspective, mentioning and putting into context the challenges that are particular to the domain, which will be the object of sections below. --->
-
+Structural bioinformatics provides scientific methods to analyse, predict, and validate the three-dimensional structure of biological macromolecules such as proteins, RNA, DNA, or carbohydrates including small molecules bound to them. It also provides an important link with the genomics and structural biology communities. One objective of structural bioinformatics is the creation of new methods of analysis and manipulation of biological macromolecular data in order to predict their structures, function and interactions. This document describes guidelines to deposit these predictions together with relevant metadata according to FAIR principles.
```

Comment:
I am not 100% sure if I understand the comment. The data needed to be kept while models are being generated is the same data that we want to have during deposition. Indeed model predictors will need to keep track of those things while they do the predictions since otherwise they may simply not have the data required for the deposition. Should I add a sentence in the intro section to make this more explicit?
E.g. "Note that predictors should collect the relevant metadata already while doing the predictions to make sure the data is available during deposition."?

Reply:
Indeed that is what I meant. A bit more details about metadata that needs to be generated while the models are being created.

Reply:
Aside from this I'm also concerned about other aspects of Structural Bioinformatics that are included here e.g. the binding site predictions using 3D models. This can be created as separate sub section.

Reply:
@smza I updated the text to clarify the need to collect data already while doing the predictions. From conversations with Sameer, it seems that this is in line with the "data harvesting" process done for experimental structures. He also commented on function predictions as being a sub section that could be added later by someone in his team. Would that be ok?

Reply:
Thanks @gtauriello. What Sameer has suggested is reasonable.